internals/cli/masker/writer.go

```diff
@@ -0,0 +1,204 @@
+package masker
```

Comment:
Could you add a short package description (godoc)?

Optionally add a larger description as well.